### ISSUE TITLE
Command confirmation 

### ISSUE THREAD

**Issue by rsmahl** [itsMeDavidV/Watchla-Support#222]
I upgraded the app with the new Tesla API, and now I have to confirm commands even though I have it unselected on both the phone and my Apple Watch

**Comment by Drummondii** [comment]
Same issue here. Please fix this issue

**Comment by kamsimon** [comment]
Same here

**Comment by zunzster** [comment]
I've finally got the new version of watchla authenticated and bluetoothed and I'm getting this too.

**Comment by itsMeDavidV** [comment]
In version 3.4.9, Watchla temporarily disabled one-tap commands out of precaution as the new UI re-write was introduced. They should now be re-enabled in the update v3.5.2 that was released hours ago.

Before testing and after updating, you'll want to make sure that your watch app has also received the update as sometimes Apple Watch apps can receive updates hours or days after their iOS companion apps do. You can check this by going to Preferences in the watch app and scrolling all the way down to the version indicator.

If you find your watch app has not updated to v3.5.2 (325) or later you can directly update it on the App Store app found on your Apple Watch.

**Comment by Drummondii** [comment]
Thanks. I can now do one tap commands. However the watch app still opens to the charge stats screen rather than the control screen. Is there a way of starting at the main co trials screen?

**Comment by zunzster** [comment]
Yes, I get that screen from the Watchla Charge status Complication too.

**Comment by Drummondii** [comment]
Well spotted @zunzster. The charge stats startup screen does seem to appear after opening the watch app from a complication rather than the app grid. (I hadn’t noticed this behaviour as I only access it from the watch face)

**Comment by zunzster** [comment]
Yeah, it would be kinda OK for the charge stats Complication to change to that if the app didn't keep freezing on that screen since it's pretty easy to cancel out of to the main Watchla screen normally.
